packages/inter-protocol/src/proposals/utils.js

```diff
@@ -90,8 +91,12 @@ export const makeInstallCache = async (
   { installCacheKey = 'installCache', loadBundle },
 ) => {
   /** @type {CopyMap<string, {installation: Installation, boardId: string, path?: string}>} */
-  const initial = await provideWhen(E.get(homeP).scratch, installCacheKey, () =>
-    makeCopyMap([]),
+
+  const initial = await provideWhen(
+    // @ts-expect-error cast
+    E.get(homeP).scratch,
```

Comment:
While I mentioned above I'd prefer to figure out the `provideWhen`'s param type to the compatible subtype, is there a reason to use `// @ts-expect-error` for casts instead of doing an actual cast ?

Do we consider casting to be too visually onerous?

Reply:
Moot with 570cc993f3cfb2eeff49a363fd10b497b100079c, but I do find casting in JSdoc to be hard to read. No strong feeling one way or the other.